internal/runnable/grpc.go

```diff
@@ -0,0 +1,52 @@
+package runnable
```

Comment:
why are we defining this internal pkg outside `pkg`?

Reply:
Yeah, probably not the best way to do it now that I see the directory structure. Moved into `pkg/ext-proc/internal`.